.strigo/config.yml

```
@@ -1,45 +1,88 @@
-## Class exercises - simple configuration for a single class
+## Class lab resources and exercises - simple configuration for a single class
 ---
 classes:
-  - exercises:
-      # `file` is relative to .strigo directory
+  - lab_resources:
+      - display_name: "AWS EC2 Instance"
+        server_image_id: "ami-0abcdef1234567890"
+        platform_type: "lab"
+        image_platform: "linux"
+        cloud_provider: "aws"
+        view_interface: "Desktop"
+        instance_type: "t2.micro"
+      - display_name: "Web Resource"
+        web_resource_url: "https://example.com"
```

Web resources have the web_resource_url, but this is a mutually exclusive fields. if web_resource_url is provided no other image related field should be provided.
so

 ```suggestion
       - lab_resources:
          - display_name: "Web Resource"
            web_resource_url: "https://example.com"
            platform_type: "web"
```

Even more importantley- we should think if this is even relevant. I think only MediaOcean are using weblabs, so for all the other users this is potentially noise and distraction.